### DISCUSSION TITLE
Being handcuffed lacks a clear prompt for new players, AI doesn't complain when handcuffed

### DISCUSSION THREAD

**Discussion by Krzeszny** [FakeFishGames/Barotrauma#9677] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I've been watching a Polish review of Barotrama from a year ago.
The reviewer makes a point that Barotrauma has glitches because one of his characters permanently lost the ability to interact with objects... because the character had been handcuffed by station security and he **never** realized what had happened, so he just switched characters.
https://www.youtube.com/watch?v=4Ck7rvzWZlY&t=395s

Someone could argue it's obvious because of handcuffs in your inventory and a lock icon... but if the inventory has many items in it, new players can miss it.

Furthermore, when bots are handcuffed, they won't say it even if you give them orders.

### Reproduction steps

1. Be a new playier
2. Get handcuffed without realizing, especially in singleplayer

### Suggested solutions
1. Add text saying "HANDCUFFED" visible on the screen when handcuffs are equipped?
2. When trying to interact with objects (and ladders) while handcuffed, add a prompt saying "HANDCUFFED", but there's a mod (Neurotrauma) that uses the handcuffs mechanic in arm dislocation, so maybe it should instead say "CANNOT USE HANDS" to be more mod-compatible.
3. When AI is handcuffed and wants to do something, the character should say "I'm in handcuffs"

### Issue prevalence

Sometimes happens ton new players

### Version

0.18.15.0

### Which operating system did you encounter this bug on?

Windows

**Comment by metakot** [comment]
Sorry, but why are you created this issue?
The video is almost 2 years old, things have changed since.
The text is visible if you hover the mouse over the inventory, also there is a hint window designed specially for newbies, if you have disabled the hints, that's totally up to you.
Handcuffs ARE VISIBLE on the ragdoll.
![image](https://user-images.githubusercontent.com/5718037/180710513-011715b4-a13b-48c7-9335-28d59b26fdfe.png)
![image](https://user-images.githubusercontent.com/5718037/180710533-a9e81291-4715-4c16-bec6-a1aeb0435353.png)

**Comment by metakot** [comment]
Lol, that explains everything =)
![image](https://user-images.githubusercontent.com/5718037/180711115-bb19a17f-23a3-42a6-9080-7eda6213c4de.png)

**Comment by Krzeszny** [comment]
> The text is visible if you hover the mouse over the inventory

That's right. The text is visible only if you hover the mouse over the inventory. If you don't know it's because of the inventory, you won't notice.

> there is a hint window designed specially for newbies, if you have disabled the hints, that's totally up to you. 

I indeed disabled hints. Not saying that most players disable hints but... they do.

> Handcuffs ARE VISIBLE on the ragdoll.

There is, NO SPRITE for handcuffs on ragdolls.
![Barotrauma_Ae14HvJxq3](https://user-images.githubusercontent.com/10529134/180824641-c6e161e3-f922-4647-85a6-19446f653aee.png)

**Comment by metakot** [comment]
@Krzeszny 
If they have disabled hints, I suspect what they are also suffocating in diving suits due to not knowing how to react to constant beeping =)
I just handcuffed my entire single player crew to show you what sprite is visible.
![image](https://user-images.githubusercontent.com/5718037/181118760-e9eee7d6-f006-4aa6-bd04-cc66f1dc0652.png)

**Comment by Krzeszny** [comment]
> I just handcuffed my entire single player crew to show you what sprite is visible.
> *image*

Then how can you explain my screenshot and why is you captain naked?

**Comment by Krzeszny** [comment]
@metakot 
I just handcuffed 4 characters in the editor and even took their clothes off to show you there is no visible sprite... Why is there a sprite in your game?
![Barotrauma_OZA6cNXamW](https://user-images.githubusercontent.com/10529134/181258656-4019d551-4824-4006-8101-b47c785bde7b.png)

**Comment by NilanthAnimosus** [comment]
vanilla has sprites for this, are you using any mods? there is both the UI and in game sprite on a vanilla copy of barotrauma running no mods, and I've yet to find any outfit with a sprite depth issue involving handcuffs?

Are you running any mods (however insignificant)?

I tested this quickly in the submarine editor:
![image](https://user-images.githubusercontent.com/29177976/181498301-07c40402-c49b-41e7-bb27-ca340b6e239b.png)
and...the handcuffs show in vanilla?

**Comment by Krzeszny** [comment]
> vanilla has sprites for this, are you using any mods?

It was Neurotrauma...

**Comment by metakot** [comment]
[Krzeszny](https://github.com/Krzeszny)
Then you should have tick the [x] "My issue happened while using mods." 
My captain is naked because I've disabled all mods despite they are not interfering in any way and his clothes got removed =D

**Reply by Krzeszny** [reply]
It's not my issue and even better, the reviewer's issue happened without mods. But I'll remove the part that says handcuffs are invisible.
